src/pages/Dashboard/index.js

```diff
@@ -140,9 +140,16 @@ function getDashboardChartConfig(state, range) {
           size: 'small',
         },
         {
-          title: 'Samples processed over time',
-          data: chartSelectors.getSamplesOverTime(state, range),
-          series: ['unprocessed', 'processed'],
+          title: 'Processed Samples over time',
+          data: chartSelectors.getProcessedSamplesOverTime(state, range),
+          series: ['total'],
+          type: 'area',
+          size: 'large',
+        },
+        {
+          title: 'Unprocessed Samples over time',
```

Comment:
Not sure if this is a good title for this chart?

```suggestion
          title: 'Created Samples over time',
```

Reply:
or maybe `Samples surveyed` to match the title below it?

Reply:
Good call probably should be Surveyed for consistency